src/runtime/virtcontainers/clh.go

```diff
@@ -474,7 +474,8 @@ func (clh *cloudHypervisor) enableProtection() error {
 		return nil
 
 	default:
-		return errors.New("This system doesn't support Confidential Computing (Guest Protection)")
+		// Allow running with no hardware protection for testing.
+		return nil
```

Comment:
[nit] would it be useful to add a log here saying there's no hardware protection?